### PR TITLE
Schedule

### PR DESCRIPTION
**Pull request by Lukasz-334**
authors: @gyrydymptum , @Lukasz-334

### CODE REVIEW

**Comment by ziobron** [comment]
✅ CI Passed - Task: schedule-bonus
🏅 Granted 10 XP for @gyrydymptum

**Comment by ziobron** [comment]
⏱ 3 XP extra for @gyrydymptum for on-time delivery

**Comment by ziobron** [comment]
🏅 +1 XP for cooperation goes to @gyrydymptum

**Comment by ziobron** [comment]
✅ CI Passed - Task: schedule-bonus
🏅 Granted 10 XP for @Lukasz-334

**Comment by ziobron** [comment]
⏱ 3 XP extra for @Lukasz-334 for on-time delivery

**Comment by ziobron** [comment]
🏅 +1 XP for cooperation goes to @Lukasz-334

**Comment by ziobron** [comment]
✅ CI Passed - Task: schedule
🏅 Granted 12 XP for @gyrydymptum

**Comment by ziobron** [comment]
⏱ 3 XP extra for @gyrydymptum for on-time delivery

**Comment by ziobron** [comment]
🏅 +1 XP for cooperation goes to @gyrydymptum

**Comment by ziobron** [comment]
✅ CI Passed - Task: schedule
🏅 Granted 12 XP for @Lukasz-334

**Comment by ziobron** [comment]
⏱ 3 XP extra for @Lukasz-334 for on-time delivery

**Comment by ziobron** [comment]
🏅 +1 XP for cooperation goes to @Lukasz-334

**Review by lukasz-kukulka — APPROVED** [review]

**Comment by ziobron** [comment]
Thanks @lukasz-kukulka for Code Review.
🏅 1 XP granted

File: homework/schedule/schedule.hpp [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+#include <thread>
+
+void schedule(std::function<void()> func, std::chrono::seconds);
+void schedule(std::function<void(int)> func, std::chrono::seconds, int);
+void schedule(std::function<void(std::string, double)> func, std::chrono::seconds, std::string, double);
+
+template <typename F, typename T, typename... Tpar>
+void schedule(F func, T waiting_time, Tpar... parameters) {
+    std::this_thread::sleep_for(waiting_time);
+    func(parameters...);
```
Comment (nauka-programowania-MA) [inline homework/schedule/schedule.hpp:13]: use perfect forwarding here. std::forward<Args>(args)...

**Review by nauka-programowania-MA — APPROVED** [review]

**Pull request closed without merging**